src/FakeItEasy.Extensions.ValueTask/FakeItEasy.Extensions.ValueTask.csproj

```diff
@@ -23,6 +23,7 @@
     <Compile Include="../FakeItEasy/ValueTaskReturnValueConfigurationExtensions.StronglyTyped.cs" />
     <Compile Include="../FakeItEasy/ValueTaskAsyncReturnValueConfigurationExtensions.cs" />
     <Compile Include="../FakeItEasy/ValueTaskAsyncReturnValueConfigurationExtensions.StronglyTyped.cs" />
+    <Compile Include="../*.cs" />

```

Comment:
I'm perhaps being silly, but could this be added to the `Directory.Build.props`, since it's copied in both `.csproj`s now?

Reply:
Sure, I can do that.